gulpfile.js

```diff
@@ -148,7 +148,9 @@ function writeEnvJson(platform, config, isRelease) {
     throw new Error(`Failed to set up environment, no such path: ${config.targetDir}`);
   }
   const envFile = `${config.targetDir}/environment.json`;
-  runCommand(`scripts/environment_json.sh -p ${platform} ${isRelease ? '-r' : ''} > ${envFile}`, {}, function () {
+  const bashPathForWindows = process.platform.includes('win') ? 'sh ' : '';
```

Comment:
`bashPathForWindows` is a misleading name if not running on Windows. How about we do something like this?

``` js
let envScript  = 'scripts/environment_json.sh';
if (process.platform.includes('win')) {
  envScript = `sh ${envScript}`;
}
```